packages/stores/types.ts

```diff
@@ -180,6 +213,61 @@ type TClientStore = {
     logout: () => Promise<LogOutResponse>;
     should_allow_authentication: boolean;
     is_crypto: boolean;
+    isEligibleForMoreDemoMt5Svg: (market_type: 'synthetic' | 'financial') => boolean;
+    isEligibleForMoreRealMt5: (market_type: 'synthetic' | 'financial') => boolean;
+    fetchResidenceList?: () => void;
+    account_settings: GetSettings & {
+        upload_file?: string;
+        poi_state?: string;
+    };
+    residence_list: ResidenceList;
+    is_high_risk: boolean;
+    should_restrict_bvi_account_creation: boolean;
+    should_restrict_vanuatu_account_creation: boolean;
+    trading_platform_available_accounts: {
+        market_type: 'financial' | 'gaming';
+        name: string;
+        requirements: {
+            after_first_deposit: {
+                financial_assessment: string[];
+            };
+            compliance: {
+                mt5: string[];
+                tax_information: string[];
+            };
+            signup: string[];
+        };
+        shortcode: 'bvi' | 'labuan' | 'svg' | 'vanuatu' | 'maltainvest';
+        sub_account_type: string;
+    }[];
+    updateMT5Status: () => void;
+    fetchAccountSettings: () => void;
+    setAccountSettings: (get_settings_response: GetSettings) => void;
+    upgradeable_landing_companies: unknown[];
+    is_populating_mt5_account_list: boolean;
+    //eslint-disable-next-line @typescript-eslint/no-explicit-any
+    landing_companies: Record<string, any>;
```

Comment:
```suggestion
    landing_companies: LandingCompanyResponse;
```
@amir-deriv I guess it should be `LandingCompanyResponse` 🤔

Reply:
```
       ERROR in src/Containers/compare-accounts-modal.tsx:136:54
       TS2339: Property 'financial_stp' does not exist on type '{}'.
           134 |             landing_companies?.mt_gaming_company?.financial,
           135 |             landing_companies?.mt_financial_company?.financial,
         > 136 |             landing_companies?.mt_financial_company?.financial_stp,
               |                                                      ^^^^^^^^^^^^^
           137 |         ];
           138 |
           139 |         const cfd_account_button_label =
           ```
           
         It doesn't look like the correct type for it @farzin-deriv

Reply:
@amir-deriv Oh seems like it should be `LandingCompany` instead of `LandingCompanyResponse` 🤔 Can you check 🙏🏻

Reply:
<img width="693" alt="image" src="https://user-images.githubusercontent.com/129206554/233292382-85494daa-7bfd-4602-a26e-11b4bc387f08.png">

nope, I don't think this is `LandingCompany`